### DISCUSSION TITLE
Passat B8 patch? MHI2_ER_VWG11_S2125_1

### DISCUSSION THREAD

**Discussion by RobertoDeGennaro** [Mr-MIBonk/M.I.B._More-Incredible-Bash#90] [Q&A] [answered]
Hello,

I have a 2016 Passat with SW MHI2_ER_VWG11_S2125_1 and I cannot find the right patch in the folder (there are only 2 patches for VWG11). Can I use one of them or it is necessary a specific one for this SW?
Thank you in advance,
Roberto

**Comment by harman-f** [comment, marked as answer]
This FW is quite old, most people update FW and patch later.

mibsolution.one

**Reply by RobertoDeGennaro** [reply]
Ok, seems the last update was made in 2019. Annualy I only update the maps.
I think updating this needs VCDS, correct?
